readthedocs/projects/forms.py

```diff
@@ -175,6 +175,29 @@ class Meta(object):
         widget=forms.Textarea,
     )
 
+    def clean_language(self):
```

Comment:
I think this is not the form you want.

`ProjectExtraForm` is used when importing a project via the Wizard. You need to write this logic in `UpdateProjectForm` which is the one used under https://readthedocs.org/dashboard/{project_slug}/edit/

Reply:
https://github.com/rtfd/readthedocs.org/blob/233febf4846d7a0aeb95b6c28962e06e21d13688/readthedocs/projects/views/private.py#L98-L111

Reply:
I was in doubt where to put it (in case it was reused on others parts). And since the `ProjectExtraForm` has the `language` field.